lostpixel.config.ts

```diff
@@ -0,0 +1,30 @@
+import { CustomProjectConfig, PageScreenshotParameter } from "lost-pixel";
+
+const defaultPageOptions: Partial<PageScreenshotParameter> = {
+    waitBeforeScreenshot: 2500
```

Comment:
Could be set on top level in lost pixel config! Page config params can help you override the top level ones 😀